### PR TITLE
updates

### PR DESCRIPTION
**Pull request by fidele000**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -357,4 +357,7 @@ build/
 *.db
 
 # Scripts
-.do_not_delete
\ No newline at end of file
+.do_not_delete
+.vscode
```
Comment (ufrshubham) [inline .gitignore:361]: Actually we need tasks.json in version control. Right now it contains a task that makes creation of new classes easier by calling create_new_class.py internally.

File: .gitignore [hunk 2]
```diff
@@ -357,4 +357,7 @@ build/
...
+build
```
Comment (ufrshubham) [inline .gitignore:362]: See line 354, build directory is already ignored.

File: .vscode/tasks.json [hunk 3]
```diff
@@ -1,31 +0,0 @@
...
-        }
-    ],
-    "inputs": [
-        {
-            "description": "Enter the class name.",
-            "id": "Class name",
-            "type": "promptString"
-        }
-    ]
-}
```
Comment (ufrshubham) [inline .vscode/tasks.json:31]: We need this file. It contains one useful task which will automate creation of new C++ classes.

File: .gitignore (hunk 1 above)
Comment (fidele000) [reply .gitignore:361]: great

File: .gitignore (hunk 2 above)
Comment (fidele000) [reply .gitignore:362]: i removed it

**Pull request merged**